source/extensions/filters/http/dynamic_forward_proxy/proxy_filter.h

```diff
@@ -51,6 +51,13 @@ class ProxyFilter
 public:
   ProxyFilter(const ProxyFilterConfigSharedPtr& config) : config_(config) {}
 
+  static const std::string& dnsStart() {
```

Comment:
nit: I think you can just do `static constexpr absl::string_view = ...` for these.